lib/types/multipart.js

```diff
@@ -169,6 +170,10 @@ function Multipart(boy, cfg) {
       else
         encoding = '7bit';
 
+      for (var h in header) {
+        if (!isRequiredHeader(h)) additionalHeaders[h] = header[h][0];
```

Comment:
I think we should not be forcing the first value all the time, in case someone is interested in seeing all of the duplicate header values. Maybe at the very least we could just return the first value if `header[h].length === 1` otherwise use the array value. Also just always assigning to `header[h]` so it's always an array for consistency is fine by me too.